tests/sdk/java/convert_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package convert
+package test
```

Comment:
Could we add a code comment somewhere to describe why we are using this namespace for the convert tests, and perhaps a todo to restructure this in the future.

Reply:
Added a `doc.go` with a link to https://github.com/pulumi/ci-mgmt/issues/676 which I think best captures the spirit of what we need to do here.